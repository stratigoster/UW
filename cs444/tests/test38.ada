-- this works
-- WARNING: another crazy test case ahead
PACKAGE foo IS
  function plus (a,b:integer) return integer; -- this returns a universal integer

  type integer is
    record
      f:float;
    end record;

  function plus (a,b:integer) return integer; -- this returns the _record_ integer
END foo;
