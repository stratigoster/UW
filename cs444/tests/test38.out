***ERROR***:0:0:	Package `MAIN' must be defined
